websites/migrations/0046_unpublish_fields.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.1.14 on 2022-04-22 16:05
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("websites", "0045_blank_publish_fields"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="website",
+            name="last_unpublished_by",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="unpublisher",
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
+        migrations.AddField(
+            model_name="website",
+            name="unpublished",
```

Comment:
The comment above in [websites/views.py](https://github.com/mitodl/ocw-studio/pull/1270/files#diff-e48ade8b030312c01525fc67ffbe535d8e900ddf191885be7fcc8d084cb6a999R277)

> `# Get all sites, minus any sites that have been unpublished or never successfully published`

made me realize that I realized I'm confused about whether `unpublished` means
1. "is currently not on live ocw", or
2. "has been removed from live ocw at some point"

I think it's (2), right?

Should we try to phrase the fields so that `is_currently_published` and `!unpublished` are equivalent?

Or...Do we need an `unpublished` db field? Is `unpublished_status` good enough?

Reply:
Ideally neither field should be updated from the admin panel, because it won't trigger the new pipeline from there, only the new API endpoint does that (`./api/<site_name>/unpublish`).  Now that I think of it, I should probably make those fields in admin uneditable.

I will go ahead and remove the `unpublished` field.  Though I think it makes the code a bit clearer in terms of what's happening, especially in query filters, it's not strictly necessary.